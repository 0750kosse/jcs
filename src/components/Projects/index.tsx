import React from 'react'
import styled from 'styled-components'
import { mediaQueries } from '../../mediaqueries'
import { ProjectProps } from '../../types'
import { projects } from '../../lib/data'
import Button from '../Button'

const Content = styled.div`
display:flex;
flex-direction:column;
align-items:center;
text-align:center;
margin:-2rem 0 4rem;
width:90%;
height:24rem;
background:white;
border-radius: 15px;
box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
${mediaQueries('tablet')`
width:45%;

margin-bottom: 6rem;
margin-top:-4rem;
`}
${mediaQueries('medium')`
margin-top:-8rem;
margin-bottom:10rem;
`}
${mediaQueries('xl')`
width:20%;
`}
`

const Header = styled.h2``
const SubHeader = styled.h3``
const Paragraph = styled.h4``

export default (props: ProjectProps) => {
  const { header, subheader, stack, demo, github } = props
  return (
    <Content className="project__wrapper--item">
      <Header>{header}</Header>
      <SubHeader>{subheader}</SubHeader>
      <Paragraph>{stack}</Paragraph>
      <Button src={demo}>Live Demo</Button>
      <Button>Github</Button>
    </Content>
  )
}