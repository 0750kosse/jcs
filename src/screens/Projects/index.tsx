import React from 'react';
import styled from 'styled-components'
import { mediaQueries } from '../../mediaqueries'
import Project from '../../components/Project'
import { projects } from '../../lib/data'

const ColToRowWrapper = styled.div`
display: flex;
flex-direction: column;
align-items:center;
${mediaQueries("tablet")`
flex-flow:row wrap; 
justify-content: space-evenly;
`} `

export default () => {
  return (
    <>
      <ColToRowWrapper className="project__wrapper" id="projects">
        {projects.map((project, index) => {
          const { header, subheader, stack, demo, github } = project
          return <Project
            key={index}
            header={header}
            subheader={subheader}
            stack={stack}
            demo={demo}
            github={github}
          />
        })
        }
      </ColToRowWrapper>
    </>
  )
}