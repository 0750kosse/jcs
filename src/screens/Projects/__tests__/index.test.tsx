import React from 'react'
import Projects from '../index'
import { render } from '@testing-library/react'

describe('Projects screen', () => {
  it('should render correclty', () => {
    const wrapper = render(<Projects />)
    expect(wrapper).toMatchSnapshot()
  })
})