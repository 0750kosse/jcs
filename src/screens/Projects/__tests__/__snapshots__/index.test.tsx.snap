// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects screen should render correclty 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-iBPRYJ gRQjDs project__wrapper"
        id="projects"
      >
        <div
          class="sc-dlfnbm cIjknv project__wrapper--item"
        >
          <h2
            class="sc-hKgILt fOCRPP"
          >
            St. Joseph´s
          </h2>
          <h3
            class="sc-eCssSg fMBprR"
          >
            A project to revamp the existing church website for a responsive, mobile-first, modern and accessible feel.
          </h3>
          <h4
            class="sc-jSgupP dQDbPF"
          >
            HTML, CSS, and Vanilla JS
          </h4>
          <div
            class="sc-gKsewC jzxyGc"
          >
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              Live Demo
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              GitHub
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
          </div>
        </div>
        <div
          class="sc-dlfnbm cIjknv project__wrapper--item"
        >
          <h2
            class="sc-hKgILt fOCRPP"
          >
            PetBuddy
          </h2>
          <h3
            class="sc-eCssSg fMBprR"
          >
            A commercial project building a responsive, writing clean, reusable code for a local pet business.
          </h3>
          <h4
            class="sc-jSgupP dQDbPF"
          >
            ReactJs, Typescript, React testing library
          </h4>
          <div
            class="sc-gKsewC jzxyGc"
          >
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              Live Demo
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              GitHub
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
          </div>
        </div>
        <div
          class="sc-dlfnbm cIjknv project__wrapper--item"
        >
          <h2
            class="sc-hKgILt fOCRPP"
          >
            Netfailx
          </h2>
          <h3
            class="sc-eCssSg fMBprR"
          >
            Bored at home...? Check out this movie search engine which uses the Open Movie Database API.
          </h3>
          <h4
            class="sc-jSgupP dQDbPF"
          >
            React, Javascript, Jest & Enzyme
          </h4>
          <div
            class="sc-gKsewC jzxyGc"
          >
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              Live Demo
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              GitHub
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
          </div>
        </div>
        <div
          class="sc-dlfnbm cIjknv project__wrapper--item"
        >
          <h2
            class="sc-hKgILt fOCRPP"
          >
            Domino ́s Pizza
          </h2>
          <h3
            class="sc-eCssSg fMBprR"
          >
            Hungry after the movie...? Then use my online takeaway pizza application...
          </h3>
          <h4
            class="sc-jSgupP dQDbPF"
          >
            Node, Express, Mongoose, AWS, MongoDB
          </h4>
          <div
            class="sc-gKsewC jzxyGc"
          >
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              Live Demo
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
            <a
              aria-label="button"
              class="sc-bdfBwQ abOIX"
            >
              GitHub
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
                data-icon="external-link-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                  fill="currentColor"
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-iBPRYJ gRQjDs project__wrapper"
      id="projects"
    >
      <div
        class="sc-dlfnbm cIjknv project__wrapper--item"
      >
        <h2
          class="sc-hKgILt fOCRPP"
        >
          St. Joseph´s
        </h2>
        <h3
          class="sc-eCssSg fMBprR"
        >
          A project to revamp the existing church website for a responsive, mobile-first, modern and accessible feel.
        </h3>
        <h4
          class="sc-jSgupP dQDbPF"
        >
          HTML, CSS, and Vanilla JS
        </h4>
        <div
          class="sc-gKsewC jzxyGc"
        >
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            Live Demo
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            GitHub
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        class="sc-dlfnbm cIjknv project__wrapper--item"
      >
        <h2
          class="sc-hKgILt fOCRPP"
        >
          PetBuddy
        </h2>
        <h3
          class="sc-eCssSg fMBprR"
        >
          A commercial project building a responsive, writing clean, reusable code for a local pet business.
        </h3>
        <h4
          class="sc-jSgupP dQDbPF"
        >
          ReactJs, Typescript, React testing library
        </h4>
        <div
          class="sc-gKsewC jzxyGc"
        >
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            Live Demo
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            GitHub
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        class="sc-dlfnbm cIjknv project__wrapper--item"
      >
        <h2
          class="sc-hKgILt fOCRPP"
        >
          Netfailx
        </h2>
        <h3
          class="sc-eCssSg fMBprR"
        >
          Bored at home...? Check out this movie search engine which uses the Open Movie Database API.
        </h3>
        <h4
          class="sc-jSgupP dQDbPF"
        >
          React, Javascript, Jest & Enzyme
        </h4>
        <div
          class="sc-gKsewC jzxyGc"
        >
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            Live Demo
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            GitHub
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        class="sc-dlfnbm cIjknv project__wrapper--item"
      >
        <h2
          class="sc-hKgILt fOCRPP"
        >
          Domino ́s Pizza
        </h2>
        <h3
          class="sc-eCssSg fMBprR"
        >
          Hungry after the movie...? Then use my online takeaway pizza application...
        </h3>
        <h4
          class="sc-jSgupP dQDbPF"
        >
          Node, Express, Mongoose, AWS, MongoDB
        </h4>
        <div
          class="sc-gKsewC jzxyGc"
        >
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            Live Demo
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="button"
            class="sc-bdfBwQ abOIX"
          >
            GitHub
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-external-link-alt fa-w-16 sc-gsTCUz gqysfZ"
              data-icon="external-link-alt"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M432,320H400a16,16,0,0,0-16,16V448H64V128H208a16,16,0,0,0,16-16V80a16,16,0,0,0-16-16H48A48,48,0,0,0,0,112V464a48,48,0,0,0,48,48H400a48,48,0,0,0,48-48V336A16,16,0,0,0,432,320ZM488,0h-128c-21.37,0-32.05,25.91-17,41l35.73,35.73L135,320.37a24,24,0,0,0,0,34L157.67,377a24,24,0,0,0,34,0L435.28,133.32,471,169c15,15,41,4.5,41-17V24A24,24,0,0,0,488,0Z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
